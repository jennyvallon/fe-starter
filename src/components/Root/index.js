import React from 'react';
import Routes from '../../pages/index';
import './index.css';

export default () => <Routes />;
