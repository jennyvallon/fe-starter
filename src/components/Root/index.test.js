import React from 'react';
import Root from './index';

test('Render the Root component', () => {
    const component = shallow(
        <Root/>
    );
    expect(component).toMatchSnapshot();
});
