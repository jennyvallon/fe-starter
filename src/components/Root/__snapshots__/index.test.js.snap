// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the Root component 1`] = `<Component />`;
